aquarius/app/assets.py

```diff
@@ -686,6 +703,15 @@ def check_required_attributes(required_attributes, data, method):
     return None, None
 
 
+def check_no_urls_in_files(base, method):
+    if 'files' in base:
+        for file in base['files']:
```

Comment:
file is a keyword in python, although this will still work, it is better to use a different name, _file, f, myfile, or whatever.